sdks/python/apache_beam/testing/load_tests/load_test_metrics_utils.py

```diff
@@ -55,6 +55,8 @@
 METRICS_TYPE_LABEL = 'metric'
 VALUE_LABEL = 'value'
 
+PUBLISH_TO_CONSOLE = True
```

Comment:
I have a feeling that this is overlooked leftover ;)

Reply:
That's true. I decided to make console default publisher.